app/components/DocumentListToolbar/DocumentListToolbar.jsx

```diff
@@ -138,56 +173,58 @@ class DocumentListToolbar extends React.Component {
           </div>
         )}
 
-        {metadata.totalCount > metadata.limit && (
-          <div className={`${styles.section} ${styles['section-pagination']}`}>
-            <Paginator
-              currentPage={metadata.page}
-              maxPages={8}
-              pageChangeHandler={pageChangeHandler}
-              totalPages={metadata.totalPages}
-            />
-
-            <div className={styles.information}>
-              <div>
-                <span
-                  className={`${styles['page-input']} ${
-                    styles['page-input-simple']
-                  }`}
-                >
-                  <form onSubmit={this.handleGoToPage.bind(this)}>
-                    <ToolbarTextInput
-                      onChange={event => {
-                        const {value} = event.target
-                        const parsedValue = Number.parseInt(value)
-
-                        if (
-                          value === '' ||
-                          (parsedValue.toString() === value && value > 0)
-                        ) {
-                          this.setState({
-                            goToPageValue: value
-                          })
-                        }
-                      }}
-                      size="small"
-                      placeholder="Go to page"
-                      value={goToPageValue}
-                    />
-                  </form>
-                </span>
+        {totalCount > limit && (
+          <div className={styles.section}>
+            <div className={styles['page-numbers']}>
+              {totalPages > 9 ? (
```

Comment:
Maybe make this `9` a variable and add a comment explaining where it comes from?

Reply:
Made it into a var; it doesn't really come from anywhere in particular, maybe just that 
```1 2 3 4 5 6 7 8 9```
is the same number of elements as 
```1 … 3 4 5 6 7 … 9```
so for this number of pages (and less) the ellipses don't save any space most of the time, but that's not important enough to put in a comment, I think